lib/compilers/pascal.ts

```diff
@@ -214,19 +238,15 @@ export class FPCCompiler extends BaseCompiler {
                 valueInBrackets = valueInBrackets.substr(1);
             }
 
-            if (isNaN(valueInBrackets)) {
```

Comment:
We were calling `isNaN` on a string

Reply:
Ok, apparently this is is actually a thing and TS just doesn't like it. I think the solution here is `Number.isNaN(Number(valueInBrackets))`.

Reply:
wow TIL